### DISCUSSION TITLE
HSR-C does not initialize to default arm position after software update to 23.04

### DISCUSSION THREAD

**Discussion by jjnak** [HumanSupportRobot/QandA#73] [Q&A] [answered]
After updating software to 23.04, HSR occasionally does not initialize to the default arm position after releasing emergency stop buttons. If we use the web based controller, we can move the joints and base position. 

In addition, even when successfully initialized to the default position, there seems to be slight displacement in the hand posutre from the desired position.

If you could provide any help on how to diagnose the issue, that would be greatly appreciated.

Thank you,

-Jun Nakanishi

**Comment by hsr-admin** [comment, marked as answer]
The occasional failure of the initial position move is a known problem since before 23.04.
We apologize for the inconvenience, but when it occurs, please release the stop button again.

The slight misalignment of the initial position posture may be due to a misaligned zero point setting.
We would like to check, was it also misaligned before the software version was upgraded?
Have you set the zero point in the past?

**Reply by jjnak** [reply]
Thank you for your reply. I understant that occasional failure in the initialization of the arm position has been a known problem before.

Regarding the slight misalignment of the initial position, I don't think we have noticed such a problem before. This time, after rebooting the robot, we did not see the issue again. We have not changed the setting of the zero point in the past. If this happenes again, I'll report to you.
